docs/digital.rst

```diff
@@ -6,6 +6,10 @@ Board-Specific Pins
 The Raspberry Pi Pico RP2040 chip supports up to 30 digital I/O pins,
 however not all boards provide access to all pins.
 
+Pin Notation
+-----------
+Keep in mind that for using Digital Pins, you need to specify the number of the pin with letter D prepended to it, e.g. D4.
```

Comment:
This is not really the case in general.  All calls in the core work at a GPIO # level.  On the official Pico boards and many others, the PCB pins are labeled directly as the GPIO pins from the chip they're attached to.  For some boards where they didn't follow this, yes you do either need to use the Dx they silk screen (assuming they actually made a full variant file w/this mapping...OTW it'll still be a 1:1 mapping) or the real GPIO it's connected to.

Maybe something like
> When using analog or digital I/Os, if you supply an integer it specifies the RP2040 GPIO pin to use.  Using `Dx` or `Ax` notation (like `D4` or `A3`) may be necessary on boards without a direct PCB pin to GPIO mapping.

Reply:
Thank you for the explanation. I used your wording with minor capitalization changes.